cmd/neofs-node/config/config.go

```diff
@@ -54,6 +55,13 @@ func New(_ Prm, opts ...Option) *Config {
 		}
 	}
 
+	if o.validate {
```

Comment:
Why conditional?

Reply:
It's made for `call_test` and `cast_test` tests that check the correct parsing of values, and there is separate configuration for tests.